models/datasets/generic/ds__appointments.sql

```diff
@@ -0,0 +1,39 @@
+select
+    p.id as patient_id,
+    p.display_id,
+    p.first_name,
+    p.last_name,
+    p.date_of_birth,
+    date_part('year', age(a.start_datetime::date, p.date_of_birth::date)) as age,
+    initcap(p.sex::text) as sex,
+    vil.id as village_id,
+    vil.name as village,
+    billing.id as billing_type_id,
+    billing.name as billing_type,
+    a.start_datetime as appointment_start_datetime,
+    a.end_datetime as appointment_end_datetime,
+    a.appointment_type_id,
+    apt.name as appointment_type,
+    a.status as appointment_status,
+    u.id as clinician_id,
+    u.display_name as clinician,
+    lg.id as location_group_id,
+    lg.name as location_group,
+    case when a.is_high_priority then 'Yes' else 'No' end as priority,
+    a.schedule_id,
+    sa.until_date::date,
```

Comment:
Should already be in date format.
```suggestion
    sa.until_date,
```